### ISSUE TITLE
Ruby 2.7 keyword arguments warning

### ISSUE THREAD

**Issue by barthez** [Shopify/graphql-batch#126]
I'm getting a warning running my project that uses latest version of graphql-batch:
```
/usr/local/bundle/gems/graphql-batch-0.4.2/lib/graphql/batch/loader.rb:14: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/app/lib/my_loader.rb:7: warning: The called method `initialize' is defined here
```

`MyLoader` initializer accepts kwargs but `Loader.for` method doesn't seem to pass them correctly to `new` method in here: https://github.com/Shopify/graphql-batch/blob/master/lib/graphql/batch/loader.rb#L14

**Comment by barthez** [comment]
Thanks @dylanahsmith, that was fast!

**Comment by michal-samluk** [comment]
Hello, can this be released as `v0.4.3`? Currently it is only available on master branch.

**Comment by dylanahsmith** [comment]
Pushed a release (https://github.com/Shopify/graphql-batch/releases/tag/v0.4.3)
